apps/src/templates/teacherDashboard/SetUpMessage.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import i18n from "@cdo/locale";
+import color from "../../util/color";
+import ProgressButton from "../progress/ProgressButton";
+
+const styles = {
+  section: {
+    width: 1075,
+    border: '3px dashed gray',
+    backgroundColor: color.white,
+    marginLeft: 35
+  },
+  heading: {
+    paddingLeft: 25,
+    paddingTop: 25,
+    fontSize: 30,
+    fontFamily: '"Gotham 5r", sans-serif',
+    color: color.teal,
+  },
+  description: {
+    paddingLeft: 25,
+    paddingTop: 25,
+    paddingBottom: 25,
+    fontSize: 18,
+    color: color.charcoal
+  },
+  button: {
+    marginLeft: 25,
+    marginBottom: 25
+
+  }
+};
+
+const SetUpMessage = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(['courses', 'sections']).isRequired
+  },
+
+  render() {
+    const { type } = this.props;
+
+    if (type === 'courses') {
```

Comment:
This seems like one of those times where you're going to tell me that I'm returning really similar things are there is a more concise way of doing this 😀 I tried something like setting variables based on the type (e.g. `if type === 'courses' heading = "Start Learning"' but I ran into what I think is a scoping problem...